pkg/kubelet/volumes.go

```diff
@@ -66,10 +64,16 @@ func (vh *volumeHost) NewWrapperMounter(volName string, spec volume.Spec, pod *a
 	}
 
 	b, err := vh.kubelet.newVolumeMounterFromPlugins(&spec, pod, opts)
```

Comment:
@screeley44 see my earlier feedback: `newVolumeMounterFromPlugins` should return a mounter and a nil error, or a nil mounter and a non-nil error, but never (nil, nil)